projects/libavif/build.sh

```diff
@@ -15,6 +15,12 @@
 #
 ################################################################################
 
+# afl++ CMPLOG test:
+test -n "$AFL_MAP_SIZE" && {
```

Comment:
So, CI fails as in non-afl builds, e.g. libfuzzer AFL_MAP_SIZE is undefined.
Can you use $FUZZING_ENGINE = "afl" for this.

Reply:
ah sure. that is better